nextjs-end/firebase.json

```diff
@@ -41,11 +41,7 @@
   },
   "hosting": {
     "source": ".",
-    "ignore": [
-      "firebase.json",
-      "**/.*",
-      "**/node_modules/**"
-    ],
+    "ignore": ["firebase.json", "**/.*", "**/node_modules/**"],
     "frameworksBackend": {
       "region": "us-central1"
     }
```

Comment:
we should probably remove the whole hosting object to reduce confusion

Reply:
There may be a web frameworks codelab floating around that uses this, so I'm leaving it in for now.